docs/how-to/configure.md

```diff
@@ -120,6 +125,8 @@ Checks use the *Standard* interval by default. To change a check's interval:
 
 (Read more at [Security Threat Tool](../using/platform/security-threat-tool.md).)
 
+If you'll disable the already enabled Security Threat Tool  feature you will not be able to see the Security alerts and Security Checks will not be executed on the system
```

Comment:
As above, we are just stating the obvious. We are in danger of insulting the intelligence of our readers with such words.